diagnostic.md

```diff
@@ -6,70 +6,70 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books/4' or to: 'books#4'
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It trasfers a copy of the previous version of the database, or from the production version. So that you can make and test new changes without affecting production, or at least the working copy.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+The scaffold commands sets up a basic rails template for the things database, where as the controller command sets up just the controller page from the database things.
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+This creates and adds the index that items in the data base will be listed by.
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```md
-// your response here
+A list of all the things that are contained under the class Patient.
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
 ```md
-// your response here
+It is defined in the patient create file.
 ```
 
 Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+Model just proecesses commands and data based on the command given. It however doesn't contain the data, and is just responding the controler.
 ```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
```

Comment:
This is a typo and should have been `bin/rails console` so no points will be deducted for this.